server/services/allocations.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
+// disabling camelcase check so that we can manipulate snake_case attributes without changing structure
+const { dbClient, collections } = require('../db');
+
+/**
+ * Gets all phases for a site
+ * @param {number} siteId PK ID of the site
+ * @param {number} phaseId PK ID of the phase
+ */
+const getAllocation = async (siteId, phaseId) => {
+  /**
+   * @typedef {Object} allocationResponse  type for DB response to the `getAllocation` query
+   * @property {number} id                 PK ID of the allocation
+   * @property {number} phase_id           FK for phase
+   * @property {number} allocation         Number of allocations available
+   * @property {number} site_id            FK for employer_site
+   *
+   *
```

Comment:
Nitpick: these extra lines can be removed.

Reply:
removed!